resource/environment/integration.js

```diff
@@ -1,49 +1,14 @@
+const _ = require( "lodash" );
+const Promise = require("bluebird");
```

Comment:
@EvanSnapp - why another promise library when we already have `when`? I'm not as familiar with `bluebird`, so I'm not against it, but I'm asking to understand. If `bluebird` has better features or support, let's discuss what those are and what it would take to change this project over. Two promise libraries is too many.